components/brave_ads/core/internal/ml/pipeline/text_processing/embedding_processing.cc

```diff
@@ -22,6 +23,21 @@
 
 namespace brave_ads::ml::pipeline {
 
+// static
+base::expected<EmbeddingProcessing, std::string>
```

Comment:
Can we please revert these changes and move back to the generic implementations for CreateFromValue until we transition these resources to protobuf/flatbuffers